### PR TITLE
Endpoints for inactive events, scores connected to eventID, get event from ID

### PR DESCRIPTION
**Pull request by selmagudmundsen**
### PR Type 🍏
What kind of change does this PR introduce?
```
[ ] Bugfix
[x] Feature
[ ] Database
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation content changes
[ ] Tests
[ ] Other
```
[Kanban Link](https://www.notion.so/bekks/742fa46fd71e47728aff33e5a32e125e?v=4fe4766ee0f54d348a595b3864658662&p=a23b1e7a1a584a50a4848a9013f49ef6&pm=s)

### Context 🤷‍♀️
We are developing the admin page further which has required us to add some endpoints in backend to fetch the correct information to the frontend. 

### What's new? 👶
- Added an endpoint to get a list of all inactive events in the database.
- Added endpoint to fetch player scores from a specific inactive event, using the event ID
- Added an endpoint to get an event from eventID.

### CODE REVIEW

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/ScoreController.kt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package org.entur.norgesturbackend.norgestur.controller
 
+import com.sun.java.accessibility.util.EventID
```
Comment (Magnus-Farstad) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/ScoreController.kt:3]: Brukes denne til noe?

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt [hunk 2]
```diff
@@ -30,4 +30,6 @@ class PlayerScoreController (val playerScoreService: PlayerScoreService){
     ): HttpStatus{
         return playerScoreService.savePlayerScore(playerScore)
     }
+
+
```
Comment (maribsta) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/controller/PlayerScoreController.kt:34]: Trengs disse ekstra mellommommene

File: src/main/kotlin/org/entur/norgesturbackend/norgestur/service/EventService.kt [hunk 3]
```diff
@@ -60,6 +70,7 @@ class EventService(
         val playersWithNoScores = playerRepository.findPlayersWithNoScores()
         playerRepository.deleteAll(playersWithNoScores)
     }
-}
 
-    class MultipleActiveEventsException(message: String) : RuntimeException(message)
\ No newline at end of file
+
+    class MultipleActiveEventsException(message: String) : RuntimeException(message)
```
Comment (Magnus-Farstad) [inline src/main/kotlin/org/entur/norgesturbackend/norgestur/service/EventService.kt:75]: Burde denne kanskje ligge i en egen mappe for exceptions?

**Review by Magnus-Farstad — APPROVED** [review]
Kjempefantastisk!

**Review by oscarahalvorsen — APPROVED** [review]
LGTM!

**Review by maribsta — APPROVED** [review]
Good work!

**Pull request merged**